doc/exits-and-hotspots.md

```diff
@@ -0,0 +1,58 @@
+# Hotspots and exits
+
+Hotspots in Escoria are thought of as "things a character can walk on", either
+like opening a door in Prince of Persia by stepping on a plate or a more abstract
+"To street" exit hotspot.
```

Comment:
Maybe it would be better to call it "trigger" to avoid confusing it with established terminology?

Reply:
So, `trigger.gd` as well.

Reply:
Myeah, I dunno what the established terminology is. In Unity/Adventure Creator iirc pretty much anything was a hotspot and additional scripts defined what kind, which is not how Godot does things.

I should have time to edit the commits in the afternoon.

Do you prefer I preserve the wording, like "Triggers in Escoria are thought of" just in case someone is as unfamiliar with established terminology as I am?

Reply:
Yeah, I think that wording is fine. :slightly_smiling_face: Maybe we should make a list of terminology to establish our use of terminology throughout the framework. I know the use of "scene" vs. "room" can also be confusing to newcomers.